docs/data/date-pickers/base-concepts/base-concepts.md

```diff
@@ -0,0 +1,144 @@
+---
+product: date-pickers
+title: Date and Time Picker - Base concepts
+packageName: '@mui/x-date-pickers'
+githubLabel: 'component: pickers'
+materialDesign: https://m2.material.io/components/date-pickers
+waiAria: https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/datepicker-dialog.html
+---
+
+# Date and Time Pickers - Base concepts
+
+<p class="description">The Date and Time pickers expose a lot of components to fit your every needs.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+## Basic usage
+
+All the components have a `value` / `onChange` API to set and control the values:
+
+{{"demo": "ControlledComponent.js"}}
+
+## Imports
+
+All the components exported by `@mui/x-date-pickers` are also exported by `@mui/x-date-pickers-pro` but without the nested imports.
+
+For example, to use the `DatePicker` component, the following three imports are valid:
+
+```tsx
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { DatePicker } from '@mui/x-date-pickers';
+import { DatePicker } from '@mui/x-date-pickers-pro';
+```
+
+## Other components
+
+### Choose interaction style
+
+Depending on your use case, different interaction styles are preferred.
+
+- For input editing with a popover for mouse interaction, use the _Picker_ components:
+
+{{"demo": "BasicDatePicker.js", "hideToolbar": true, "bg": "inline"}}
+
+- For input-only editing, use the _Field_ components:
+
+{{"demo": "BasicDateField.js", "hideToolbar": true, "bg": "inline"}}
+
+- For inline editing, use the _Calendar / Clock_ components:
```

Comment:
Shouldn't this demo the `Static` picker version? 🤔 
Separate sub-components could maybe be a different section on it's own?

Reply:
I don't think so

This section says "a picker is a field plus one or several views"
And it demonstrates, the picker, the field and the view.

If I start talking about the static pickers, who are pickers but not really because I said just before that a picker had a field and the static pickers don't have one, I fear it will confuse everybody.

Reply:
Mhh, fair point. I did not look at it from this PoV.
Sound fair in that case. 👍